vite_ruby/lib/vite_ruby/build.rb

```diff
@@ -45,14 +46,15 @@ def retry_failed?
   end
 
   # Internal: Returns a new build with the specified result.
-  def with_result(success)
+  def with_result(success, err_msg=nil)
```

Comment:
Let's make it required:

```suggestion
  def with_result(success, errors)
```

Reply:
if we're okay with kw args, i'd make the second param there a keyword arg too, if not both of them. Why use mysterious positional args when you can use self-documenting kw args.